test/test.el

```diff
@@ -79,6 +79,10 @@
   (should (equal (floor (ts-unix (ts-now)))
                  (string-to-number (format-time-string "%s")))))
 
+(ert-deftest ts-internal ()
+  (should (and (consp (ts-internal (ts-now)))
+               (not (proper-list-p (ts-internal (ts-now)))))))
```

Comment:
If we're going to test this, we should probably ensure that each element is an integer, as intended.  I'd probably use `pcase`.